src/handle-data.ts

```diff
@@ -95,9 +95,7 @@ export function getColumnInfo(layout: TableLayout, colIdx: number, pageColIdx: n
   const { qDimensionInfo, qMeasureInfo } = layout.qHyperCube;
   const numDims = qDimensionInfo.length;
   const isDim = colIdx < numDims;
-  const info = (isDim ? qDimensionInfo[colIdx] : qMeasureInfo[colIdx - numDims]) as
-    | ExtendedNxMeasureInfo
-    | ExtendedNxDimensionInfo;
```

Comment:
Are they supposed to be removed ?

Reply:
The typecasting was apparently not needed.